src/Altinn.AccessManagement/appsettings.json

```diff
@@ -13,7 +13,7 @@
   },
   "PostgreSQLSettings": {
     "EnableDBConnection": "true",
-    "WorkspacePath": "Migration",
+    "WorkspacePath": "Altinn.AccessManagement.Persistence\\Migration",
```

Comment:
Path needs to be fixed

Reply:
Fixed in latest commit. Updated the dockerfile and moved the WorkspacePath for dev to appsettings.Development.json